tests/integration/test_reload_certificate/test.py

```diff
@@ -65,7 +66,7 @@ def change_config_to_key(name, pass_phrase=""):
             ),
         ]
     )
-    node.query("SYSTEM RELOAD CONFIG")
```

Comment:
Sleep is not ideal, why is it required now?

Reply:
I need to test auto-reloading without manually running the command.

Reply:
I'm a little bit concerned with this being a flaky check. But ok, let's try 2 seconds. We can figure out something later if it becomes flaky